Projects/Contracts/Sources/Screens/MovingFlow/MovingFlowIntro.swift

```diff
@@ -120,7 +115,7 @@ extension MovingFlowIntro: Presentable {
 
         let activeContractBundles: Future<[GraphQL.ActiveContractBundlesQuery.Data.ActiveContractBundle]> =
             client.fetch(
-                query: GraphQL.ActiveContractBundlesQuery(),
+                query: GraphQL.ActiveContractBundlesQuery(locale: Localization.Locale.currentLocale.asGraphQLLocale()),
```

Comment:
We should pass in the bundle we are moving directly into moving-flow I think, instead of fetching it again here.